src/sentry/static/sentry/app/views/settings/projectAlerts/projectAlertHeader.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SentryTypes from 'app/sentryTypes';
+import Button from 'app/components/button';
+import ListLink from 'app/components/listLink';
+import NavTabs from 'app/components/navTabs';
+
+import Tooltip from 'app/components/tooltip';
+
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+
+import withOrganization from 'app/utils/withOrganization';
```

Comment:
These should all be grouped

Reply:
We should have a lint rule for this, otherwise I'm not gonna overthink import order